stream-projects/notifications/truffle.config.mjs

```diff
@@ -0,0 +1,254 @@
+      filters: {
+        route: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+    {
+      filters: {
+        eventTopic: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+    {
+      filters: {
+        eventCron: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+    {
+      filters: {
+        action: { isAll: true, rank: 0 },
+      },
+      action: "create",
+      value: true,
+    },
+    {
+      filters: {
+        action: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+    {
+      filters: {
+        eventSubscription: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+    {
+      filters: {
+        notificationTopic: { isAll: true, rank: 0 },
+      },
+      action: "create",
+      value: true,
+    },
+    {
+      filters: {
+        notificationTopic: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+    {
+      filters: {
+        notificationJob: { isAll: true, rank: 0 },
+      },
+      action: "create",
+      value: true,
+    },
+    {
+      filters: {
+        keyValue: { isAll: true, rank: 0 },
+      },
+      action: "update",
+      value: true,
+    },
+  ],
+  installActionRel: {
+    // a workflow run a list of actions
+    actionPath: "@truffle/core@latest/_Action/workflow",
+    runtimeData: {
+      mode: "sequential", // sequential | parallel
+      // create event topic that will be broadcast an event every 5 minutes
+      // to check if the streamer is live
+      stepActionRels: [
+        {
+          actionPath: "@truffle/core@latest/_Action/graphql",
+          runtimeData: {
+            query: gql`
+              mutation EventTopicUpsert($input: EventTopicUpsertInput!) {
+                eventTopicUpsert(input: $input) {
+                  eventTopic {
+                    id
+                  }
+                }
+              }
+            `,
+            variables: {
+              input: {
+                resourcePath: `${PACKAGE}/_EventTopic/notify-is-live`,
+              },
+            },
+          },
+        },
+        // the event cron that will actually trigger the broadcast every 5 minutes
+        {
+          actionPath: "@truffle/core@latest/_Action/graphql",
+          runtimeData: {
+            query: gql`
+              mutation EventCronUpsert($input: EventCronUpsertInput) {
+                eventCronUpsert(input: $input) {
+                  eventCron {
+                    id
+                    cronExpression
+                  }
+                }
+              }
+            `,
+            variables: {
+              input: {
+                resourcePath: `${PACKAGE}/_EventCron/notify-is-live`,
+                eventTopicPath: `${PACKAGE}/_EventTopic/notify-is-live`,
+                // TODO: switch this to 5 minutes
```

Comment:
think every min is actually fine for this. users complain about the YT notifications taking too long to deliver

Reply:
Update: https://github.com/trufflehq/truffle-packages/pull/49/commits/1e09c858a3d40f72f0e13c20efc241c6e7fa246d